src/System.ComponentModel.Composition/src/Microsoft/Internal/Requires.cs

```diff
@@ -95,6 +95,26 @@ public static void IsInMembertypeSet(MemberTypes value, string parameterName, Me
                 throw new ArgumentException(string.Format(CultureInfo.CurrentCulture, SR.ArgumentOutOfRange_InvalidEnumInSet, parameterName, value, enumFlagSet.ToString()), parameterName);
             }
             Contract.EndContractBlock();
-        }        
+        }
+
+        public static void NotNull<T>(T value, string parameterName)
```

Comment:
are you still using this file? We also want to get rid off this one too if possible as it is just a fork.

Reply:
I copied two methods from Common Microsoft.Internal Required.cs to the local Required.cs in System.ComponentModel.Composition to avoid tons of extra changes. Still should i replace these validation with inline checks?

Reply:
Ok cool. I think for the sake of this PR this is good. Maybe we can address that remaining work with a follow-up PR 👍